### ISSUE TITLE
Feature for refreshing beacons

### ISSUE THREAD

**Issue by ItsAdityaKSingh** [CCExtractor/beacon#91]
**Is your feature request related to a problem? Please describe.**
Users can't fetch recently activated beacons as there is no way of refreshing screen.

**Describe the solution you'd like**
Adding a pull to refresh option for beacons.

**Additional context**

https://user-images.githubusercontent.com/69353350/150695008-4e17d479-174a-4b45-a6fd-ddd60f3ae7ce.mov

**Are you working on this? (Yes/No)**
Yes

**Comment by codewithkushagra** [comment]
Is this issue resolved if not can I work on it? @ItsAdityaKSingh

**Comment by Rushour0** [comment]
Even I want to work on this if it is not been worked upon @ItsAdityaKSingh

**Comment by ItsAdityaKSingh** [comment]
Yes, sure! You may work upon this feature and propose changes :)

**Comment by Rushour0** [comment]
@ItsAdityaKSingh do check my latest PR

**Comment by ItsAdityaKSingh** [comment]
Yes, sorry for the delay! Facing academics, would review it soon asap :)

**Comment by lalitkumawat1m** [comment]
Hi, could you please assign this issue to me?
I would love to work on it and contribute to the project.

**Comment by azizur-rahaman** [comment]
hello, i want to work on this issue, can i ?
